petab/parameter_mapping.py

```diff
@@ -58,12 +60,22 @@ def get_optimization_to_simulation_parameter_mapping(
             Whether parameter values should be scaled.
 
     Returns:
-        The length of the returned array is n_conditions, each entry is a tuple
-        of two dicts of length n_par_sim, listing the optimization parameters
-        or constants to be mapped to the simulation parameters, first for
-        preequilibration (empty if no preequilibration condition is specified),
-        second for simulation. ``NaN`` is used where no mapping exists.
+        Parameter value and parameter scale mapping for all conditions.
+
+        The length of the returned array is the number of unique combinations
```

Comment:
```suggestion
        The lengths of the returned array are respectively the numbers of unique combinations
```

Reply:
I'd say it was correct before. It's an array of tuples, not a tuple of arrays.

Reply:
of course, you are right!